src/Interactive/VbiCore/vbi.rsp

```diff
@@ -0,0 +1,35 @@
+/r:System.Collections.dll
```

Comment:
Exactly, what does it mean to have VBI/CSI core taking references in a response file? For example, you're passing System.Runtime, which in the CoreCLR folder, has type forwards to mscorlib, which you don't reference.

ie Where exactly is this rsp being used, and have you tested it?